### DISCUSSION TITLE
How to use  field inversion

### DISCUSSION THREAD

**Discussion by 15175829310** [mdolab/dafoam#600] [General]
How can I invert the physical quantities of several specific points instead of the entire flow field for flow field inversion

**Comment by friedenhe** [comment]
Check the template from: https://github.com/mdolab/dafoam/blob/695e280b3bb38ebd07635e09cd66369162302148/tests/runTests_DAPimpleFoamField.py#L78

**Reply by 15175829310** [reply]
Thank you for your reply. I'll look at this file
